pom.xml

```diff
@@ -242,9 +236,14 @@
             </dependency>
 
             <dependency>
-                <groupId>org.slf4j</groupId>
-                <artifactId>slf4j-log4j12</artifactId>
-                <version>${slf4j-log4j12.version}</version>
+                <groupId>org.apache.logging.log4j</groupId>
+                <artifactId>log4j-core</artifactId>
+                <version>${apache.log4j.version}</version>
+            </dependency>
```

Comment:
Why do we need a dependency on log4j-core? It should not be necessary, only log4j-slf4j-impl should be enough to get logging going.

Reply:
thanks you are right. Shows my lack of understanding of SLF4j. I will fix it.